src/Gridify/GridifyMapper.cs

```diff
@@ -220,6 +220,39 @@ public IEnumerable<IGMap<T>> GetCurrentMaps()
    {
       return _mappings;
    }
+   
+   public IEnumerable<IGMap<T>> GetCurrentMapsByType(HashSet<Type> targetTypes)
+   {
+      foreach (var map in _mappings)
+      {
+         if (map.To.Body is UnaryExpression unaryExpression)
```

Comment:
minor improvement, we can convert this to a switch statement (not important)

Reply:
Done